### ISSUE TITLE
Validate radio button

### ISSUE THREAD

**Issue by Nick-StormID** [stormid/components#232]
Adding "required" validation adds _aria-required_ to the inputs, which fails [WCAG21 labels-or-instructions](https://www.w3.org/TR/WCAG21/#labels-or-instructions).
Need to manage "required" validation on a radiogroup without adding those attributes.

**Comment by Nick-StormID** [comment]
Covered with https://github.com/stormid/components/issues/233
